contracts/TalentLayerEscrow.sol

```diff
@@ -896,12 +925,18 @@ contract TalentLayerEscrow is
      */
     function _reimburse(uint256 _transactionId, uint256 _amount) private {
         Transaction storage transaction = transactions[_transactionId];
-        uint256 totalReleaseAmount = _calculateTotalWithFees(
+        uint256 reimbursedReferralAmount;
```

Comment:
4 lines can be compact into 1: (just add a commentary: if there is no referrer, referralAmount is 0, reimbursedReferralAmount will be 0)

uint256 reimbursedReferralAmount = (_amount * transaction.referralAmount) / (transaction.totalAmount);